docs/presentation/index.html

```diff
@@ -84,24 +90,24 @@
 
           <section>
             <h2>Light Metrics</h2>
-            <p class="fragment">intensity</p>
-            <p class="fragment">wavelength</p>
-            <p class="fragment">correlated color temperature</p>
+            <p class="fragment">Intensity</p>
+            <p class="fragment">Wavelength</p>
+            <p class="fragment">Correlated Color Temperature</p>
```

Comment:
As far as I can recall, it was never discussed adding. Does CCT replace PPFD? The structure that was proposed is here https://github.com/njason/plant-recipe-schema/blob/master/examples/tomatofest/tomatofest-full-recipe.xml#L20-L41. Is the structure you are proposing:

```
<light>
  <band min="450" max="500">
    <ppfd min="0" max="0" />
    <cct min="0" max="0" />
  </band>
</light>
```

Reply:
The comment intended from before..

No.. CCT doesn't replace PPFD. Using band/wavelength makes sense when we're talking about specific color channels. For example:
UV-A at 380 nm
Blue at 450 nm
Red at 650 nm
Far Red at 730 nm

CCT takes into account if a grower is also using white light, whether it's from LEDs which you'd see in smaller vertical farming setups, or High Pressure Sodium (HPS) lamps which you'd commonly see for indoor greenhouses. Basically, CCT is how warm or cool the light is, given off either a slightly amber/red hue (like sunlight in the early morning post sunrise), or slightly blue hue (like sunlight around noon, without cloud cover). The units used are in K, and typically range from 2400K (Warm Lighting), to 7400K (sunny blue sky). Because it more closely resembles natural sunlight, specifying a wavelength doesn't make any sense, since the spectrum is pretty broad. It's much more similar to the spectrum images we have listed for this slide.

I'd included CCT in the Growth Recipe Doc a while ago, and I had brought it up in past meetings.. but it's definitely possible, I may have spoken on it, but it got missed somehow. 

This all said, just with narrow bandwidth color channels using nanometers as their unit, and CCT using Kelvin as its unit.. both of the two still specify PPFD as the unit for intensity. So that aspect translates over just fine. 

The structure that I'd propose is that user could specify either band:
<band min="450" max="500">
<ppfd min="50" max="50" />
</band>

or CCT
<cct min="2700" max="2800">
<ppfd min="50" max="50" /> 
</cct>

for a <light></light> section of a phase.